### ISSUE TITLE
I need to click a button to activate a JS function multiple times

### ISSUE THREAD

**Issue by Trynight** [MetaMask/metamask-mobile#3970]
To activate a function from a button, I need to click several times before the metamask app identifies the JS link.

**To Reproduce*
You can try it yourself from the Metamask App (Android), by going to one of these 2 sites:

private.everbusd.finance
nft.xape.app 

**Expected behavior**
It should work immediately after the first click, but you need to do more than one (4-5) before the button goes

**Smartphone (please complete the following information):**
 - Device: Xiaomi MI 11
 - OS: Android
 - App Versionv4.2.2(834)

**Comment by gantunesr** [comment]
Hey @Trynight, can you check if this issue occurs in another mobile browsers?

**Comment by Trynight** [comment]
> Hey @Trynight, can you check if this issue occurs in another mobile browsers?

No, I didn't have try another mobile browsers... cause as i know metamask as estentions doesn't work on mobile on ff/chrome.
